src/components/Check/Check.scss

```diff
@@ -10,25 +10,37 @@
 $checkBoxHeight: 20px;
 
 .ms-Check {
-  display: inline-block;
-  cursor: default;
-  vertical-align: top;
-  width: $checkBoxHeight;
-  height: $checkBoxHeight;
+  font-family: FabricMDL2Icons;
+  line-height: 1;
   text-align: center;
+  width: 18px;
+  height: 18px;
+  vertical-align: top;
   position: relative;
+  cursor: default;
```

Comment:
it seems odd that we'd add a cursor here. It's this just an icon glyph component? Wouldn't we want to control cursor at the layer above? Is it broken right now?